tracepoint_decode/src/perf_event_format.rs

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+extern crate alloc;
+
+use core::fmt;
+
+use alloc::string;
+use alloc::vec;
+
+use crate::*;
+use perf_field_format::ascii_to_u32;
+use perf_field_format::consume_string;
+use perf_field_format::is_space_or_tab;
+
+/// At present, does nothing.
+macro_rules! debug_eprintln {
```

Comment:
... this is what the logging crate(s) are for.

I think it's OK for crates like this to have more dependencies, especially on massively popular and common crates like "log". It even has best practices for libraries to follow (link to "log" and use the macros, but don't pick your own logger implementation) so that downstream consumers can enable messages from your library at-will.

Reply:
I don't disagree, but this is just a placeholder for a case where I'm not sure what I want to do. In Windows, this might be a good place for a telemetry assert.